src/VSIX_Dev17/VSIX_Dev17.csproj

```diff
@@ -0,0 +1,81 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Paul Harrington.  All Rights Reserved.  Licensed under the MIT License.  See LICENSE in the project root for license information. -->
+<Project>
+  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
+
+  <PropertyGroup>
+    <TargetFramework>net472</TargetFramework>
```

Comment:
💡 This is possible without having two separate VSIX. I implemented an example in https://github.com/laurentkempe/GitDiffMargin/pull/401 and could do similar here.

Reply:
I thought it might be, but I'm following the guidance on https://docs.microsoft.com/en-us/visualstudio/extensibility/migration/update-visual-studio-extension?view=vs-2022#use-shared-projects-for-multi-targeting
I don't want to deviate too far from the official guidance until I've got something stable.
(At the moment, I'm running into an odd case where **both** VSIXs get installed into the Dev17 experimental instance... which is bizarre since the original clearly doesn't support Dev 17!)

Reply:
> (At the moment, I'm running into an odd case where both VSIXs get installed into the Dev17 experimental instance... which is bizarre since the original clearly doesn't support Dev 17!)

The eventual fix for this will allow setting DeployExtension to false in all cases, and then deploying just the appropriate extension(s) for integration testing and/or local debugging.

Reply:
The doc says that the marketplace doesn't currently accept multi-targeting VSIXes and recommends publishing to a preview extension.